### PR TITLE
accented

### PR DESCRIPTION
**Pull request by msadowsky**
@kbenker 
Adding a function for accents

### CODE REVIEW

File: udfs.sql [hunk 1]
```diff
@@ -778,3 +778,19 @@
...
       );
     
+
+      /*
+      REMOVE_ACCENTS
+      Remove accents from a varchar
+
+      Examples:
+      'André' --> 'Andre'
+      'más' --> mas
```
Comment (kbenker) [inline udfs.sql]: missing single quotes around result.

otherwise, this all LGTM. did you create it already?

Comment (msadowsky) [inline udfs.sql]: Result?

Yeah, I created it. It works except when there are apostrophes in the string or if it's null. I probably should update this to deal with these issues before merging.

Comment (kbenker) [inline udfs.sql]: 'más' --> mas
second one should be 'mas'

Comment (msadowsky) [inline udfs.sql]: Ah. Gotcha.

**Pull request merged**